assets/embauche.html

```diff
@@ -5,53 +5,51 @@
+					<option value="établissement" disabled>Mon administration</option>
 				</select>
 
 				de
 				<input name="effectif_entreprise" id="effectif_entreprise" type="number" min="0" placeholder="1" value="1" max="99999">
 				<label for="effectif_entreprise" title="En équivalents temps pleins : un mi-temps vaut 0,5, par exemple.">salariés</label>
 
-				&nbsp;<!-- non-breakable space to prevent minification from mangling all space with preceding <label> -->souhaite rémunérer un.e
+				&nbsp;<!-- non-breakable space to prevent minification from mangling all space with preceding <label> -->souhaite rémunérer un·e
 				<select data-provides="employee">
-					<option value="salarié.e">salarié.e</option>
-					<option value="apprenti">apprenti.e</option>
+					<option value="salarié·e">salarié·e</option>
+					<option value="apprenti">apprenti·e</option>
 					<option value="stagiaire" disabled>stagiaire</option>
 					<option disabled>aide à domicile</option>
-					<option disabled>intermittent.e du spectacle</option>
+					<option disabled>intermittent·e du spectacle</option>
 				</select>
 
 				<input type="hidden" name="contrat_de_travail_debut" value="2015-10-01"/>
 
-				en statut
+				en&nbsp;statut
 				<select name="type_sal">
 					<option value="prive_non_cadre">non-cadre</option>
 					<option value="prive_cadre">cadre</option>
 				</select>
 
 				à hauteur de
-				<input name="salaire_de_base" id="salaire" type="number" min="0" max="9999999" placeholder="1457,52" value="1457.52" step="any"><label for="salaire">&nbsp;€</label>
-
-				<select data-sets="#salaire@name">
-					<option value="salaire_de_base">brut</option>
-					<option disabled value="salaire_net_a_payer">net</option>
-				</select>
-
-				<select hidden name="allegement_fillon_mode_recouvrement">
-					<option value="progressif">progressive</option>
-					<option value="fin_d_annee">en fin d'année</option>
-					<option value="anticipe_regularisation_fin_de_periode" selected="selected">anticipée</option>
-				</select>
-
-				par
-				<select data-provides="period">
-					<option disabled>an</option>
-					<option>mois</option>
-					<option disabled>jour</option>
-					<option disabled>heure</option>
-				</select>
+				<fieldset>
+					<input name="salaire_de_base" id="salaire" type="number" min="0" max="9999999" placeholder="1457,52" value="1457.52" step="any">
+					<label for="salaire">
+						&nbsp;€
+					</label>
+					<span class="input-help">Rémunération totale, dont primes. <br>Temps plein <em>(min. 1457 €)</em> ou partiel. </span>
+					
+					<select data-sets="#salaire@name">
+						<option value="salaire_de_base">brut</option>
+						<option disabled value="salaire_net_a_payer">net</option>
+					</select>
 
+					<span>par</span>
+					<select data-provides="period">
+						<option disabled>an</option>
+						<option>mois</option>
+						<option disabled>heure</option>
+					</select>
+				</fieldset>
+				<br>
```

Comment:
Double line break or orphan input on small screens, leading to dropouts. If it's a touch device, the user can sometimes miss scrolling if there is no visible content beneath. Not major, but as a matter of principle using `<br>` should always be used as a last resort.